test/simplecov_test_helper.rb

```diff
@@ -0,0 +1,16 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+
+# The minimum version of Ruby required (by New Relic) to run SimpleCov
+# NOTE: At the time that SimpleCov was introduced to this code base, it
+# required Ruby >= 2.5.0 and Ruby 2.6.0 was marked for EOL
```

Comment:
The original comment suggested for here contained an accidental typo (I think I had `SImple` instead of `Simple`). Seeing if you would remove it wasn't meant to be a test, but if it was you and your attention to detail would have passed! Nicely done.